src/Makefile.in

```diff
@@ -56,8 +65,9 @@ ifeq ($(VERILATOR_NO_OPT_BUILD),1)  # Faster laptop development... One build
 	-mv -f $@.tmp $@
 else
 ../bin/verilator_bin$(EXEEXT): obj_opt ../bin prefiles
-	$(MAKE) -C obj_opt -j 1  TGT=../$@ -f ../Makefile_obj serial
-	$(MAKE) -C obj_opt       TGT=../$@ -f ../Makefile_obj
+	$(RM_OBJ_OPT_COMPILE_COMMANDS)
```

Comment:
But if the submakes do nothing, does this break?

Reply:
I moved this comment to: https://github.com/verilator/verilator/pull/4463.